src/renderer/components/settings/settings.css

```
@@ -11,4 +11,62 @@
 
 .vcd-settings-title {
     margin-bottom: 0.5rem;
-}
\ No newline at end of file
+}
+
+.tray-icon-wrap {
```

all classes should be prefixed
```suggestion
.vcd-tray-icon-wrap {
```